localisation/00_wc_sad.csv

```diff
@@ -770,7 +772,53 @@ nick_twilight_of_the_gods;the Twilight of the Gods;;die G
 recruit_old_god_guard;Raise Old God's Guard;;Die Garde des alten Gottes ausheben;;;;;;;;;;;x
 recruit_old_god_guard_desc;The Old Gods can create legions of servants from their aberrant flesh and release them upon any fools who dare to face them.;;Die alten G�tter k�nnen Legionen von Dienern aus ihrem abartigen Fleisch erschaffen und sie auf jeden Narren loslassen, der es wagt, sich ihnen zu stellen.;;;;;;;;;;;x
 ;;;;;;;;;;;;;;x
+become_brewmaster;Become a Brewmaster;;Werdet ein Braumeister.;;;;;;;;;;;x
+become_brewmaster_desc;Learn the secrets of making powerful brews.;;Erlernt die Geheimnisse der Herstellung starker Biere.;;;;;;;;;;;x
+halaani_whiskey_intoxication;Halaani Whiskey Intoxication;;Halaani-Whisky-Rausch;;;;;;;;;;;x
+halaani_whiskey_intoxication_desc;Consumers find themselves talking a lot.;;Die Konsumenten sind oft redselig.;;;;;;;;;;;x
+sulfuron_slammer_intoxication;Sulfuron Slammer Intoxication;;Sulfuronwasser-Rausch;;;;;;;;;;;x
+sulfuron_slammer_intoxication_desc;It's gonna knock your socks off, then set your feet on fire.;;;;;;;;;;;;;x
+metal_brew_intoxication;Metal Brew Intoxication;;Metallbr�u-Rausch;;;;;;;;;;;x
+metal_brew_intoxication_desc;It holds the heat of a burning forge with the strong, stern tones of the earth.;;;;;;;;;;;;;x
+tiger_stout_intoxication;Tiger Stout Intoxication;;Tigerbr�u-Rausch;;;;;;;;;;;x
+tiger_stout_intoxication_desc;Gives the consumer the swiftness of the tiger. Also makes them prone to meowing.;;;;;;;;;;;;;x
+serpent_ale_intoxication;Serpent Ale Intoxication;;Schlangenbr�u-Rausch;;;;;;;;;;;x
+serpent_ale_intoxication_desc;It's admirers often blabble about ssssecret knowledge.;;;;;;;;;;;;;x
```

Comment:
I'd say, intoxication desc should speak of the effect it already had on you, not what effect you'd get when you drink again.